jax/_src/numpy/ufuncs.py

```diff
@@ -253,9 +253,54 @@ def bitwise_count(x: ArrayLike, /) -> Array:
   # Following numpy we take the absolute value and return uint8.
   return lax.population_count(abs(x)).astype('uint8')
 
-@implements(np.right_shift, module='numpy')
 @partial(jit, inline=True)
 def right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  r"""Right shift the bits of ``x1`` to the amount specified in ``x2``, element by
+  element. If x1.shape != x2.shape, they must be compatible for broadcasting to a
+  shared shape, this shared shape will also be the shape of the output. Right shifting
+  a scalar x1 by scalar x2 is equivalent to :code:`x1 // 2**x2`.
```

Comment:
Use double back-ticks rather than a `:code:` directive: it's more readable as plain text.